src/shared/components/Loaders/FullPageLoader.tsx

```diff
@@ -28,7 +60,13 @@ export default function FullPageLoader({ loaded }: { loaded: boolean }) {
       >
         <GoldenCircleSpinner />
         <LoadingText>
-          <p>Loading your experience...</p>
+          <p>
+            Loading your experience...{" "}
+            {getLoadingText(
+              loaderType,
+              walletOnboarded !== null && isConnected
```

Comment:
is it possible to have `isConnected=true` but  `walletOnboarded=false`? Reading this code it seems like `isConnected` would be enough

Reply:
It is possible, because `walletOnboarded` value changes when user enters the portal. So there could be a situation when user connected the wallet, but did not enter the portal and without this condition additional loading text would be displayed on onboarding loader (when entering onboarding page) and I think we only want this additional text to be displayed when entering the map